compiler/rustc_metadata/src/rmeta/mod.rs

```diff
@@ -445,6 +445,10 @@ const TAG_VALID_SPAN_LOCAL: u8 = 0;
 const TAG_VALID_SPAN_FOREIGN: u8 = 1;
 const TAG_PARTIAL_SPAN: u8 = 2;
 
+// Tags for encoding Symbol's
+const SYMBOL_STR: u8 = 0;
+const SYMBOL_OFFSET: u8 = 1;
```

Comment:
Should we use a special-purpose enum?

Reply:
I left as it is, there already big bunch of similar tags around:
https://github.com/rust-lang/rust/blob/acfbd3b320a99aabc0f35827148244e052dce01c/compiler/rustc_query_impl/src/on_disk_cache.rs#L31-L44